misoc/integration/builder.py

```diff
@@ -169,7 +169,17 @@ def _initialize_rom(self):
                     w = boot_file.read(4)
                     if not w:
                         break
+                    if self.soc.cpu_type == "vexriscv":
+                        # Little endian swap
+                        w = w[::-1]
                     boot_data.append(struct.unpack(">I", w)[0])
```

Comment:
If it *really* has to be done here, ``struct.unpack`` can convert endianness already, just use ``<I``.